app/models/piece.rb

```diff
@@ -3,11 +3,33 @@ class Piece < ActiveRecord::Base
 	belongs_to :game
 
 
-	#Allows an easier navigation between model. E.g.  Rook.find(56) => <Rook id: 56, name: :rook, color: :white, image: nil, position: 00, game_id: 20...>
+  #Allows an easier navigation between model. E.g.  Rook.find(56) => <Rook id: 56, name: :rook, color: :white, image: nil, position: 00, game_id: 20...>
   scope :rooks, -> { where(type: 'Rook') } 
   scope :knights, -> { where(type: 'Knight') } 
   scope :bishops, -> { where(type: 'Bishop') }
   scope :kings, -> {where(type: 'King') }
   scope :queens, -> {where(type: 'Queen') }
   scope :pawns, -> {where(type: 'Pawn') }
+
+  def move_to!(x_axis, y_axis)#Check methods .same_piece() and .capture()
+  	if same_piece(x_axis, y_axis).nil? 
+      update_attributes(:x_axis => x_axis, :y_axis => y_axis)
+  	elsif self.color == same_piece(x_axis, y_axis).color
+      print "Traitor!"
```

Comment:
let's make this case, where we are unable to move, `return false`.  All other cases should `return true`.